1-resilient-messaging/README.md

```diff
@@ -8,6 +8,7 @@ W celu uruchomienia warsztatu będzie nam potrzebny jedynie [Docker](https://doc
 
 # Instalacja
 
+0. Otwórz konsolę komend i przejdź do folderu w którym jest ten plik README.md. Pamiętaj, że wszystkie komendy `docker-compose` zadziałają tylko wtedy, gdy wykonasz je będąc w folderze w którym istnieje plik docker-compose.yml
 1. Uruchom komendę `docker-compose pull && docker-compose up -d`
```

Comment:
@dgafka co myślisz, żebyśmy w instrukcji zmienili komendę `docker-compose up `-d na komendę która pokaże wszystkie logi? Myślę, że może to ułatwić sprawdzanie dlaczego u kogoś może nie działać odpalenie projektu, jeśli takie przypadki się pojawią. W wersji -d, nie widać żadnych logów więc użytkownik nawet nie wie czy coś się odpaliło

Reply:
Oki, poprawisz na `docker-compose pull && docker-compose up` ?